src/threadpoolwork-inl.h

```diff
@@ -24,13 +24,16 @@
 
 #if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
 
-#include "util-inl.h"
 #include "node_internals.h"
+#include "tracing/trace_event.h"
+#include "util-inl.h"
 
 namespace node {
 
 void ThreadPoolWork::ScheduleWork() {
   env_->IncreaseWaitingRequestCounter();
+  TRACE_EVENT_NESTABLE_ASYNC_BEGIN0(
+      TRACING_CATEGORY_NODE1(threadpool), "", this);
```

Comment:
I would find the generic "thread poll" category somewhat dull as it doesn't tell us what work it is synchronously executing.

It would be more helpful to indicate what exact thread poll work it is in the tracing events if we are going to trace it.

Reply:
How about replacing the `""` with `Blob`, `zlib`, `crypto` and `node_api` ?

Reply:
Yeah, tracing with a name sounds good to me.

Reply:
Thanks ! Done !